djvm/build.gradle

```diff
@@ -148,10 +152,18 @@ shadowJar {
 }
 assemble.dependsOn shadowJar
 
+def securityPolicy = file('permissive.policy')
+
 tasks.withType(Test) {
     systemProperty 'deterministic-rt.path', configurations.jdkRt.asPath
     systemProperty 'sandbox-libraries.path', configurations.sandboxTesting.asPath
 
+    // A security policy that allows known code-bases to do anything.
+    systemProperty 'java.security.policy', "=${securityPolicy.toURI()}"
```

Comment:
Why do we need these `=` when setting the policy file?

Reply:
Setting `java.security.policy` without a preceding `=` character _appends_ the new policy to the base security policy that is installed with the JRE. But the preceding `=` creates the following option instead:
```
-Djava.security.policy==<uri-for-security-policy>
```
This replaces the security policy _completely_, which is also why the policy file needs to include this block (copied from the base policy):
```
grant codeBase "file:${{java.ext.dirs}}/*" {
    permission java.security.AllPermission;
};
```